src/main/java/cloud/eppo/ConfigurationRequester.java

```diff
@@ -0,0 +1,50 @@
+package cloud.eppo;
```

Comment:
Not sure why git is no longer pick up on the filename change

Reply:
weird!

Reply:
Git data model doesn't have a notion of renames (in git commit, it's one file deleted and another one added). So whenever a file is deleted, git applies a heuristic to find the file if it was renamed (and maybe slightly modified). With more changes added, it looks like we're past git's modification threshold for it to be considered a rename